### ISSUE TITLE
[web-ui] "Add" and "Delete" buttons are inverted on tables and databases.

### ISSUE THREAD

**Issue by Pupix** [rethinkdb/rethinkdb#5344]
Hello! Today I've encountered a UX problem which almost made me lose an entire DB. The `add/delete` buttons are inverted between table and database, as shown below:

![rethink](https://cloud.githubusercontent.com/assets/4929965/12755319/386be8d4-c9cf-11e5-8d50-b70dd4aa768d.png)

This is a very counter intuitive layout. 

If there are more DBs the `Delete selected tables` button will go out of view leaving you at a glance with `Add tables` and `Delete ...` which naturally makes your brain fill in the blank with `Delete tables` when in fact it's `Delete database`.

![rethink 2](https://cloud.githubusercontent.com/assets/4929965/12755572/45c133ee-c9d0-11e5-82a7-b8e27a3c5913.png)

Cheers.

**Comment by deontologician** [comment]
Sorry you ran into this! Now that it's pointed out, I agree this is really dumb. Some of these buttons were added later, so that might be why it happened like this.

Should be a pretty quick fix

**Comment by danielmewes** [comment]
Yeah. The `Delete ...` buttons are relatively safe because there is going to be a second confirmation step if you accidentally click them. But I agree that this isn't intuitive.

Also mentioning director of UX @mglukhovsky .

**Comment by mglukhovsky** [comment]
I believe these were changed in the web UI rewrite (correct, @deontologician?) -- we originally hid these under an "operations" button to avoid this kind of confusion.

Two different ways we could improve this:
  - make the header sticky, so as you scroll past, you still see the header with the global buttons
  - move "delete selected tables" to each database, and have it appear only once you start checking off items

@danielmewes: do both `delete` buttons include a second confirmation step?

**Comment by deontologician** [comment]
I thought there was a PR that added one of these buttons, but I can't find it so maybe I was thinking of something else. The biggest issue is that delete database is next to add table and add database is next to delete tables. So we could also switch it so that the table operations are in the database block, and the database operations are at the top. This would probably require a checkbox for the database blocks, or a way to select the right database in the modal

**Comment by danielmewes** [comment]
@mglukhovsky
> do both delete buttons include a second confirmation step?

Yes. The "Delete selected tables" button lists the tables in a modal and requires an additional confirmation click.
The "Delete database" button actually makes you enter the database name before you can continue.

Since the issue is about accidentally deleting the database instead of the selected tables, things should be very safe in practice because of the confirmation step that database deletion involves.

Still we should probably change this of course.
